client/blb/client_test.go

```diff
@@ -135,6 +166,22 @@ func testWriteRead(t *testing.T, blob *Blob, length int, off int64) {
 	}
 }
 
+// testWrite does a single write at the given length and offset.
+func testWrite(t *testing.T, blob *Blob, length int, off int64) {
+	blob.Seek(off, os.SEEK_SET)
+	p1 := makeData(length)
+	checkWrite(t, blob, p1)
+}
+
+// testRead tests a single read at a given length and offset.
+func testRead(t *testing.T, blob *Blob, len int, off int64, done chan<- bool) {
```

Comment:
also, testRead and testWrite don't really add that much on top of the existing checkRead and checkWrite. can we just combine their functionality so there aren't two very similar sets of functions?